lang/en_us.json

```diff
@@ -207,5 +207,13 @@
   "upgrade.tardis_refined.tardis_xp": "System Upgrades",
   "upgrade.tardis_refined.tardis_xp.description": "Allows upgrades to the TARDIS",
   "upgrade.tardis_refined.waypoints": "Waypoints",
-  "upgrade.tardis_refined.waypoints.description": "Allows the Pilot to create saved locations"
+  "upgrade.tardis_refined.waypoints.description": "Allows the Pilot to create saved locations",
```

Comment:
We use Forge's data providers to generate these lang files. Please refer to the lang provider on how to write these translatable strings.

Reply:
Aha was wondering how the lang stuff worked, fixed👍